CMakeLists.txt

```
@@ -2183,7 +2183,7 @@ curl_add_if("TLS-SRP"       USE_TLS_SRP)
 curl_add_if("HTTP2"         USE_NGHTTP2)
 curl_add_if("HTTP3"         USE_NGTCP2 OR USE_QUICHE OR USE_MSH3 OR USE_OPENSSL_QUIC)
 curl_add_if("MultiSSL"      CURL_WITH_MULTI_SSL)
-curl_add_if("HTTPS-proxy"   _ssl_enabled AND (USE_OPENSSL OR USE_GNUTLS
+curl_add_if("HTTPS-proxy"   NOT CURL_DISABLE_PROXY and _ssl_enabled AND (USE_OPENSSL OR USE_GNUTLS
```

```suggestion
curl_add_if("HTTPS-proxy"   NOT CURL_DISABLE_PROXY AND _ssl_enabled AND (USE_OPENSSL OR USE_GNUTLS
```
Uppercasing `AND` fixes it.

---

oh, thanks!